docs/ref/clvm.md

```diff
@@ -125,24 +125,23 @@ And in this example, an atom that was read in as an integer is appended to a str
 
 ### Atoms as Byte Arrays
 
-The atom is treated as an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
+The atom is an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
 
-### Unsigned Integer
```

Comment:
there are no unsigned integers in clvm. All Arithmetic operators interpret atoms as signed integers. The only exception is `lsh`, but it could be argued that's not an arithmetic operator. I think it's misleading to talk about unsigned integers.